paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/PaymentSheetTest.kt

```diff
@@ -339,4 +344,39 @@ internal class PaymentSheetTest {
 
         page.clickPrimaryButton()
     }
+
+    @Test
+    fun testOxxoPayment() = runPaymentSheetTest(
+        resultCallback = ::assertCompleted,
+    ) { testContext ->
+        networkRule.enqueue(
+            host("api.stripe.com"),
+            method("GET"),
+            path("/v1/elements/sessions"),
+        ) { response ->
+            response.testBodyFromFile("elements-sessions-mx-requires_payment_method.json")
+        }
+
+        testContext.presentPaymentSheet {
+            presentWithPaymentIntent(
+                paymentIntentClientSecret = "pi_example_secret_example",
+                configuration = PaymentSheet.Configuration(
+                    merchantDisplayName = "MX Mobile Example Account",
+                    allowsDelayedPaymentMethods = true,
+                ),
+            )
+        }
+
+        page.waitForText("OXXO")
+        page.clickViewWithText("OXXO")
+
+        // Only testing that the buy button is disabled initially and enabled after entering
```

Comment:
We could possibly use espresso intents to test that the webview is "launched"?

https://developer.android.com/reference/androidx/test/espresso/intent/Intents#intending(org.hamcrest.Matcher%3Candroid.content.Intent%3E)

https://developer.android.com/training/testing/espresso/intents